sympy/utilities/tests/test_lambdify.py

```diff
@@ -460,6 +469,24 @@ def test_numexpr_userfunctions():
     func = lambdify((x, y), uf(x, y), modules='numexpr')
     assert numpy.allclose(func(a, b), 2*a*b+1)
 
+def test_tensorflow_basic_math():
+    if not tensorflow:
+        skip("tensorflow not installed.")
+    expr = Max(sin(x), Abs(1/(x+2)))
+    func = lambdify(x, expr, modules="tensorflow")
+    a = tensorflow.constant(0, dtype=tensorflow.float32)
+    s =  tensorflow.Session()
+    assert func(a).eval(session=s) == 0.5
+
+def test_tensorflow_placeholders():
+    if not tensorflow:
+        skip("tensorflow not installed.")
```

Comment:
The Sage build runs on the newer ubuntu. If I remember correctly it slows down the build, so we don't want to do it for every build.  So for now I would just add `TEST_TENSORFLOW="true"` to the `TEST_SAGE="true"` build and add a `./bin/test -k tensorflow`.  Does tensorflow support Python 3 yet? We may need to move that build to use Python 2.7 (shouldn't matter; the Sage tests use the Sage Python). 

The only downside: that build is an allowed failure, because it fails sometimes (I don't remember why). So we will need to check it manually before merging this PR, and any future ones.